vrecord

```diff
@@ -1124,6 +1140,17 @@ else
     done
 fi
 
+if [[ "${embed_choice}" != "${undeclaredoption}" ]] ; then
+    echo ""
```

Comment:
?

Reply:
I was copying the syntax of similar existing options (like framemd5_choice) and didn't stop to think on that line - probably unnecessary/worth changing in all of those?